### PR TITLE
Readme link update

### PR DESCRIPTION
**Pull request by polar-kev**
Updated the "Getting Started" link associated with the "Getting started with MRTK" image
Updated the Release Assets link to point to MRTK 2.4 release assets instead of 2.0 release assets.

## Overview
As part of the MRTK Github.io docs changes, there's a new getting started page for MRTK. The URL was updated to reflect this.

## Changes
Changed Getting started link to point to https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/WelcomeToMRTK.html

Release Assets now points to: https://github.com/microsoft/MixedRealityToolkit-Unity/releases/tag/v2.4.0

### CODE REVIEW

**Comment by keveleigh** [comment]
@polar-kev Should this go into the 2.5 release branch?

**Comment by polar-kev** [comment]
@keveleigh yes, these changes should go into the 2.5 prerelease branch as well. Right now the dev branch shows by default on the MRTK github page which is why I pointed these changes there initially.

File: README.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ You can find  other example scenes under [**Assets/MixedRealityToolkit.Examples/
...
-You can find pre-built app packages for HoloLens(x86), HoloLens 2(ARM), and Windows Mixed Reality immersive headsets(x64) under [**Release Assets**](https://github.com/microsoft/MixedRealityToolkit-Unity/releases/tag/v2.0.0) folder. [Use the Windows Device Portal to install apps on HoloLens](https://docs.microsoft.com/hololens/hololens-install-apps#use-the-windows-device-portal-to-install-apps-on-hololens). On HoloLens 2, you can downloand and install [MRTK Examples Hub through the Microsoft Store app](https://www.microsoft.com/en-us/p/mrtk-examples-hub/9mv8c39l2sj4).
+You can find pre-built app packages for HoloLens(x86), HoloLens 2(ARM), and Windows Mixed Reality immersive headsets(x64) under [**Release Assets**](https://github.com/microsoft/MixedRealityToolkit-Unity/releases/tag/v2.4.0) folder. [Use the Windows Device Portal to install apps on HoloLens](https://docs.microsoft.com/hololens/hololens-install-apps#use-the-windows-device-portal-to-install-apps-on-hololens). On HoloLens 2, you can downloand and install [MRTK Examples Hub through the Microsoft Store app](https://www.microsoft.com/en-us/p/mrtk-examples-hub/9mv8c39l2sj4).
```
Comment (keveleigh) [inline README.md:81]:
```suggestion
You can find pre-built app packages for HoloLens(x86), HoloLens 2(ARM), and Windows Mixed Reality immersive headsets(x64) under [**Release Assets**](https://github.com/microsoft/MixedRealityToolkit-Unity/releases/tag/v2.4.0) folder. [Use the Windows Device Portal to install apps on HoloLens](https://docs.microsoft.com/hololens/hololens-install-apps#use-the-windows-device-portal-to-install-apps-on-hololens). On HoloLens 2, you can download and install [MRTK Examples Hub through the Microsoft Store app](https://www.microsoft.com/en-us/p/mrtk-examples-hub/9mv8c39l2sj4).
```

Comment (keveleigh) [reply README.md:81]: downloand -> download

>On HoloLens 2, you can downloand and install [MRTK Examples Hub through the Microsoft Store app]

**Comment by keveleigh** [comment]
@polar-kev Looks like a merge conflict :/

**Comment by polar-kev** [comment]
@keveleigh resolved

**Review by keveleigh — APPROVED** [review]

**Pull request merged**